keyboards/planck/keymaps/3geek14/keymap.c

```diff
@@ -0,0 +1,246 @@
+    *                      v------------------------RGB CONTROL--------------------v
+    * ,-----------------------------------------------------------------------------------.
+    * |      |Qwerty|      |RgbTog|RgbMod| HUE+ | HUE- | SAT+ | SAT- |BRGTH+|BRGTH-|      |
+    * |------+------+------+------+------+------+------+------+------+------+------+------|
+    * |      |Colemk|Music+| AudOn|AudOff|      |      |      |      |      |      | Boot |
+    * |------+------+------+------+------+------+------+------+------+------+------+------|
+    * |      |Voice-|Voice+| MusOn|      |MidiOn|MidiOf|      |      |      |      |      |
+    * |------+------+------+------+------+------+------+------+------+------+------+------|
+    * |      |      |      |      |      |             |      | Vol0 | Vol- | Vol+ |RgbTog|
+    * `-----------------------------------------------------------------------------------'
+    */
+    [_ADJUST] = LAYOUT_planck_grid(
+        _______, QWERTY,  _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______,
+        _______, COLEMAK, MU_NEXT, AU_ON,   AU_OFF,  _______, _______, _______, _______, _______, _______, QK_BOOTLOADER,
+        _______, AU_PREV, AU_NEXT, MU_ON,   _______, MI_ON,   MI_OFF,  _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, XXXXXXX, _______, KC_MUTE, KC_VOLD, KC_VOLU, RGB_TOG
+    ),
+
+    /* Utils
+    * ,-----------------------------------------------------------------------------------.
+    * |      |      |      |      |  O.O |      | (+1) |   7  |   8  |   9  |      |dashes|
+    * |------+------+------+------+------+------+------+------+------+------+------+------|
+    * |      |      |PrtScn|      |      |      |  hi  |   4  |   5  |   6  |      |      |
+    * |------+------+------+------+------+------+------+------+------+------+------+------|
+    * |      | Undo |      |      |      |      |   0  |   1  |   2  |   3  |   ‽  |      |
+    * |------+------+------+------+------+------+------+------+------+------+------+------|
+    * |      |C+PgUp|C+PgDn|Scn <-|Scn ->|             |      | email|  sig |shrug?|      |
+    * `-----------------------------------------------------------------------------------'
+    */
+    [_UTILS] = LAYOUT_planck_grid(
+        _______, _______,          _______,            EYES,        _______,      _______, THUMB_U, KC_7,    KC_8,  KC_9,      _______, DASHES,
+        _______, _______,          SCREENSHOT,         _______,     _______,      _______, WAVE,    KC_4,    KC_5,  KC_6,      _______, _______,
+        _______, LCTL(KC_Z),       _______,            _______,     _______,      _______, KC_0,    KC_1,    KC_2,  KC_3,      INTERRO, _______,
+        _______, LCTL(KC_PAGE_UP), LCTL(KC_PAGE_DOWN), SCREEN_LEFT, SCREEN_RIGHT, _______, XXXXXXX, _______, EMAIL, SIGNATURE, SHRUG,   _______
+    )
+};
+// clang-format on
+
+#ifdef AUDIO_ENABLE
```

Comment:
you don't need this code.  `set_single_persistent_default_layer(x)` actually handles that, with a define.
https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Eg, you want to add something like this to your config.h:
```c
#define DEFAULT_LAYER_SONGS { SONG(QWERTY_SOUND), \
                              SONG(COLEMAK_SOUND) \
                            }
```

Reply:
Ah, thanks! Done.